apps/service_providers/llm_service/default_models.py

```diff
@@ -73,6 +73,10 @@ def k(n: int) -> int:
         Model("llama-3.1-8b-instruct", 131072),
         Model("llama-3.1-70b-instruct", 131072),
     ],
+    "deepseek": [
+        Model("deepseek-v3", 128000),
+        Model("deepseek-r1", 128000)
```

Comment:
These should be `deepseek-chat` and `deepseek-reaoner`: https://api-docs.deepseek.com/quick_start/pricing